tests/models/test_queries.py

```diff
@@ -21,13 +21,13 @@ def create_tagged_query(self, tags):
         return query
 
     def test_all_tags(self):
-        self.create_tagged_query(tags=['tag1'])
```

Comment:
All the changes here are to convert strings to unicode to avoid SQLA warnings (unrelated to the PR).